src/app/components/wrapper-components/part-creation/part-creation.component.spec.ts

```diff
@@ -275,7 +276,8 @@ describe('PartCreationComponent:', () => {
         it('should update the form data when changing first player', fakeAsync(async() => {
             testUtils.clickElement('#firstPlayerOpponent');
 
-            expect(component.configFormGroup.get('firstPlayer').value).toEqual(FirstPlayer.CHOSEN_PLAYER.value);
+            expect(Utils.getNonNullable(component.configFormGroup.get('firstPlayer')).value)
+                .toEqual(FirstPlayer.CHOSEN_PLAYER.value);
```

Comment:
prefer
const var: type = choseLongue;
expect(var).toEqual(thingy)

to
expect(choseLongue)
....toEqual(thingy)

?

Reply:
done